tests/Simplify/ListTest.elm

```diff
@@ -1517,13 +1517,45 @@ a = List.member d (b :: c :: d :: eToZ)
 a = True
 """
                         ]
-        , test "should not report List.member d (a :: b :: c :: dToZ)" <|
+        , test "should not report List.member e (a :: b :: c :: dToZ)" <|
             \() ->
                 """module A exposing (..)
-a = List.member e (b :: c :: d :: eToZ)
+a = List.member e (a :: b :: c :: eToZ)
```

Comment:
We don't use the letter `a` there because `a` already refers to the name of the declaration and we don't want to create a recursive declaration here.